### PR TITLE
Fixes for SF3.4

### PR DESCRIPTION
**Pull request by erik-zicht**
TODO: create a new releasebranch when approved and update README and CHANGELOG

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,9 +19,8 @@
         "doctrine/orm": "*",
         "phpunit/phpunit": "^5.7",
         "sonata-project/admin-bundle": "*",
-        "symfony/symfony": "^2.7",
-        "twig/twig": "^1",
-        "zicht/standards-php": "^3"
+        "symfony/symfony": "^3.4",
+        "twig/twig": "^1"
```
Comment (pbergman) [inline composer.json]: should be twig:^2 because of php version

**Review by pbergman — CHANGES_REQUESTED** [review]

**Comment by pbergman** [comment]
there is also an pr to release/7.x open, and think it would be better to merge this in 8 based on 7?

**Comment by erik-zicht** [comment]
Thats fine with me, can this PR be merged?

**Comment by erik-zicht** [comment]
Or can we both merge these PRs in release 7 ?

**Review by pbergman — APPROVED** [review]

**Pull request merged**